### ISSUE TITLE
I can't find where the negative log-likelihood  function is used in the code

### ISSUE THREAD

**Issue by LT156** [jacobswan1/ViTCAP#11]
`
def forward(self, logits, target):
        self.iter += 1
        if logits.numel() == 0:
            # this happens when we ignore masked tokens for unmatched
            # image-text pairs, in which we may ignore all tokens
            return torch.tensor(0., requires_grad=True, device=logits.device)
        eps = self.label_smoothing
        n_class = logits.size(1)
        one_hot = torch.zeros_like(logits).scatter(1, target.view(-1, 1), 1)
        one_hot = one_hot * (1 - eps) + (1 - one_hot) * eps / (n_class - 1)
        log_prb = self.log_soft(logits)
        loss = self.kl(log_prb, one_hot).sum(1)
        if self.drop_worst_ratio > 0 and self.iter > self.drop_worst_after:
            loss, _ = torch.topk(loss,
                    k=int(loss.shape[0] * (1-self.drop_worst_ratio)),
                    largest=False)
        loss = loss.mean()
        return loss
`
In this class "class BertCaptioningLoss(nn.Module):"
I think this may be caption loss Lcap("minimizing the negative log-likelihood over the masked tokens "),
 but here is a kl loss, I cant't understand.
